shard.yml

```diff
@@ -1,5 +1,5 @@
 name: openapi-generator
-version: 2.0.0
+version: 2.1.0
```

Comment:
Could you revert the version bump? PRs should not modify this, it should be done only before releasing on master.